packages/amplify-cli/src/cli.js

```diff
@@ -6,6 +6,17 @@ const globalPrefix = require('./lib/global-prefix');
 
 const MIGRATE = 'migrate';
 
+// Gluegun has no hooks for errors happening inside executing the commands,
+// so we need a process level handler, to log the errors like environment
+// not initialized and such errors in a friendly way without technical data
+// like stacktrace which does not add value in this case.
+process.on('uncaughtException', handleError);
+
+function handleError(error) {
```

Comment:
Will this show some messages twice?

Reply:
It is only for uncaught errors, generally speaking: no, but we do have a few places in our ```catch``` blocks where we ```console.log``` and ```throw``` afterwards, but cannot be determined if there is a higher catch for those blocks or not.

IMHO refactoring the error handling to be true SEH is a different issue, that should be visited in a separate PR for the new platform.